kazoo/testing/__init__.py

```diff
@@ -1,6 +1,5 @@
 from kazoo.testing.harness import KazooTestCase, KazooTestHarness
 
-
```

Comment:
Interesting, I thought this was supposed to be two lines after imports, I'm surprised hound didn't complain... Maybe I'm wrong though?

Reply:
I think you are right, so I just re-added even if Hound seemed to be happy